pkg/registry/graph.go

```diff
@@ -0,0 +1,34 @@
+package registry
+
+type Package struct {
+	Name           string
+	DefaultChannel string
+	Channels       []Channel
+}
+
+type Channel struct {
+	Name            string
+	OperatorBundles []OperatorBundle
+	Head            BundleRef
+}
+
+type OperatorBundle struct {
+	BundlePath      string
+	Version         string // semver string
+	CsvName         string
```

Comment:
In some newer parts of this codebase, we have moved away from `CsvName` to `BundleName` or `Identifier`

Reply:
I think the intention of this type is to explicitly be an aggregate identifier. At some point `CsvName` could be deprecated entirely. For now they are equivalent though, and I think it's more likely that this field would be removed than refer to a different concept. Wdyt?